Worker/Program.cs

```diff
@@ -4,39 +4,73 @@
 using System.Threading.Channels;
 using static System.Console;
 bool start = true;
-bool back = false;
-//WriteLine("Witaj czy chcesz utworzyć swoją listę zakupów ?");
-//WriteLine("Naciśnij Enter jeśli chcesz przejść dalej");
+List<object> list = new List<object>();
```

Comment:
Is there any point to use object here instead of Articles?? Why is this list defined here? Shouldn't it be defined inside exampleList?